stl/inc/condition_variable

```diff
@@ -90,46 +111,47 @@ public:
     template <class _Lock, class _Rep, class _Period, class _Predicate>
     bool wait_for(_Lock& _Lck, const chrono::duration<_Rep, _Period>& _Rel_time, _Predicate _Pred) {
         // wait for signal with timeout and check predicate
-        return _Wait_until1(_Lck, chrono::steady_clock::now() + _Rel_time, _Pred);
+        return wait_until(_Lck, chrono::steady_clock::now() + _Rel_time, _STD move(_Pred));
     }
 
-    template <class _Lock, class _Clock, class _Duration>
-    cv_status wait_until(_Lock& _Lck, const chrono::time_point<_Clock, _Duration>& _Abs_time) {
-        // wait until time point
-        for (;;) {
-            const auto _Now = _Clock::now();
-            if (_Abs_time <= _Now) {
-                return cv_status::timeout;
-            }
+    template <class _Lock>
+    [[deprecated("This extension is deprecated and WILL BE REMOVED.")]] cv_status wait_until(

```

Comment:
Let's pull these `xtime` deprecations for now and come back with something more comprehensive. (And on 124.)